extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -123,6 +123,8 @@ export function updatedToSdkStyleError(projectName: string) { return localize('u
 export const enterNewName = localize('enterNewName', "Enter new name");
 //#endregion
 
+export const reservedProjectFolders = ['Properties', 'SQLCMD Variables', 'Database References'];
```

Comment:
Should these folder names be blocked at the DacFx level?

Reply:
I was thinking about that, but `SQLCMD Variables` and `Database References` are only reserved folders in ADS, so might not make sense to block using those folder names if someone else uses the DacFx.Projects library and isn't using ADS to view their project